app/design/adminhtml/Magento/backend/Magento_Bundle/web/css/source/_module.less

```diff
@@ -0,0 +1,15 @@
+// /**
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
+
```

Comment:
Please remove extra blank lines, add a blank line at the end of the file and fix the formatting (indents)